lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2325,6 +2357,16 @@ function EDITOR:AC_Check( notimer )
 			self:AC_SetVisible( false )
 			return
 		end
+		
+		if suggestions[1].str( suggestions[1] ) == "vec" or suggestions[1].str( suggestions[1] ) == "vec4" then
```

Comment:
Understanding your code is hard too, there's so many functions scattered around, The one thing is, the suggestion table is all automated so I couldn't find a way to insert an option there without messing it up.